hack/release.sh

```diff
@@ -272,10 +272,13 @@ release_binaries() {
 	# TODO create redirect from builds/*/i686 to builds/*/i386
 
 	cat <<EOF | write_to_s3 s3://$BUCKET_PATH/builds/index
-# To install, run the following command as root:
-curl -sSL -O $(s3_url)/builds/Linux/x86_64/docker-$VERSION.tgz && sudo tar zxf docker-$VERSION.tgz -C /
+# To install, run the following commands as root:
+curl -sSL -O $(s3_url)/builds/Linux/x86_64/docker-$VERSION.tgz && tar zxf docker-$VERSION.tgz
```

Comment:
Perhaps even nicer to use `--strip-components`, wdyt?

    curl -sSL -O $(s3_url)/builds/Linux/x86_64/docker-$VERSION.tgz && tar --strip-components=1 -xvzf docker-$VERSION.tgz -C /usr/local/bin